### PR TITLE
feat(harpoon)!: remove "j" keymap, make "t" keymap dynamic based on $TMUX

### PR DESCRIPTION
**Pull request by RayJameson**
I believe it's better when you have to remember one less keybinding.
Hope nobody gets upset because of the removed "j" keymap.
I may add it for compatibility

### CODE REVIEW

**Comment by RayJameson** [comment]
randomly fixed the error that occurred if used canceled input when using the "go to terminal" thing, also added a new keymap to jump to the index of the mark

**Comment by luxus** [comment]
anyone else that uses harpoon can confirm that?

**Comment by RayJameson** [comment]
@luxus on second thought, maybe it's better to leave keymaps as they do not contradict each other. You can still make `:term` while using TMUX, maybe someone using it. And in my experience, the TMUX keymap doesn't work as intended, I checked the plugin repo there is at least one open issue:
- thePrimeagen/harpoon#180

I can remove remap part and leave everything else

**Review by mehalter — COMMENTED** [review]
Hey @RayJameson just a heads up, I went ahead and rebased this on `main` because there was a bit of standardization that was done for filenames

**Comment by owittek** [comment]
> This also fixes which key preview, by adding an empty function. Before it was `<prefix><prefix>`, now it's `󱡀 Harpoon`, however I didn't figure out how to make it blue color as it should be.

Could you please elaborate what the empty function fixes exactly? Having any functionality (like an anonymous function) leads to the which-key entry not being colored.

File: lua/astrocommunity/motion/harpoon/init.lua [hunk 1]
```diff
@@ -7,27 +20,35 @@ return {
...
+    { "<C-p>", function() require("harpoon.ui").nav_prev() end, desc = "Goto previous mark" },
+    { "<C-n>", function() require("harpoon.ui").nav_next() end, desc = "Goto next mark" },
+    { prefix .. "m", "<cmd>Telescope harpoon marks<CR>", desc = "Show marks in Telescope" },
     {
-      prefix .. "j",
+      prefix .. "t",
       function()
-        local num = tonumber(vim.fn.input "GoTo Tmux window number: ")
-        require("harpoon.tmux").gotoTerminal(num)
+        if running_tmux_seesion then
```
Comment (mehalter) [inline lua/astrocommunity/motion/harpoon/init.lua:41]:
```suggestion
        if running_tmux_session then
```

Propagate typo fix

**Review by mehalter — CHANGES_REQUESTED** [review]
A few small comments about more easily setting variables and small typo fix

**Review by mehalter — APPROVED** [review]
LGTM!

**Comment by mehalter** [comment]
@RayJameson I think the last thing before we merge, is do you think you could come up with a good way to word the fact that there are binding changes from this commit? I think that would take higher precedence than the dynamic changing of the text in the message

**Comment by RayJameson** [comment]
@mehalter done

**Comment by mehalter** [comment]
Thanks for your work on this @RayJameson ! ❤️

**Pull request merged**